### PR TITLE
Creating credit card directly to FTX

### PR DESCRIPTION
**Pull request by gastonri**
**Description**
Previously, the backend was in charge to interact with FTX to create a credit card. Now we're moving that responsibility to the frontend to avoid sending sensitive data through our server.
- Adding a library and a helper to encrypt data (openpgp).
- Changing POST endpoint from our backend to FTX. Also, the request body changed. 
- Removing 400 HTTP status code from error handling to avoid unnecessary logout.
- Showing error from FTX to the user.

**Reviewers:** 
@Alejoboga20 @natokra @kevin-rodriguez @ImanolH96 

There is no visual changes.

### CODE REVIEW

File: src/infrastructure/components/Select/InputSelect.tsx [hunk 1]
```diff
@@ -27,6 +38,7 @@ export const InputSelect = ({ className, label = '', name, error, register, opti
 
 interface InputSelectProps {
   options: Option[];
+  alpha3Code?: boolean;
```
Comment (Alejoboga20) [inline src/infrastructure/components/Select/InputSelect.tsx:41]: also, can we rename this prop as isAlpha3Code? just to keep our convention

File: src/infrastructure/services/creditCard/CreditCardService.ts [hunk 2]
```diff
@@ -29,13 +49,22 @@ const creditCardApi = api.injectEndpoints({
         method: 'DELETE',
       }),
     }),
+    getPublicKeys: builder.mutation<PublicKeysResult, void>({
```
Comment (Alejoboga20) [inline src/infrastructure/services/creditCard/CreditCardService.ts:52]: WDYT about removing this as it is no needed anymore?

File: src/infrastructure/services/creditCard/CreditCardService.ts [hunk 3]
```diff
@@ -89,5 +121,6 @@ export const {
       matchRejected: getCreditCardByIdRejected,
     },
     deleteCreditCard: { matchFulfilled: deleteCreditCardFulfiled },
+    getPublicKeys: { matchFulfilled: getPublicKeysFulfilled },
```
Comment (Alejoboga20) [inline src/infrastructure/services/creditCard/CreditCardService.ts:124]: same here as matchers are only used for request that interact with state.

File: src/presentation/components/KYC/KYC.tsx [hunk 4]
```diff
@@ -63,6 +63,7 @@ export const KYC = () => {
               name="country"
               error={errors.country}
               options={countries}
+              alpha3Code
```
Comment (Alejoboga20) [inline src/presentation/components/KYC/KYC.tsx:66]: rename this prop if previous change is applied

**Review by Alejoboga20 — COMMENTED** [review]
Great Job. Just a few comments

File: src/infrastructure/components/Select/InputSelect.tsx [hunk 5]
```diff
@@ -2,7 +2,15 @@ import { MenuItem, Select, Typography } from '@material-ui/core';
...
 
-export const InputSelect = ({ className, label = '', name, error, register, options }: InputSelectProps) => {
+export const InputSelect = ({
+  className,
+  label = '',
+  name,
+  error,
+  register,
+  options,
+  alpha3Code,
```
[the remark this answers is not in the document]
Comment (gastonri) [reply src/infrastructure/components/Select/InputSelect.tsx:12]: You mean isAlpha3Code or alpha3Code? The first one is the boolean and the second one is the country code.
If it is isAlpha3Code, I agree.

File: src/presentation/components/KYC/KYC.tsx (hunk 4 above)
Comment (gastonri) [reply src/presentation/components/KYC/KYC.tsx]: Done!

File: src/infrastructure/services/creditCard/CreditCardService.ts (hunk 3 above)
Comment (gastonri) [reply src/infrastructure/services/creditCard/CreditCardService.ts]: Done!

File: src/infrastructure/services/creditCard/CreditCardService.ts (hunk 2 above)
Comment (gastonri) [reply src/infrastructure/services/creditCard/CreditCardService.ts]: Yes, you're right. Already removed!

File: src/infrastructure/components/Select/InputSelect.tsx (hunk 1 above)
Comment (gastonri) [reply src/infrastructure/components/Select/InputSelect.tsx]: Of course. Renamed!

File: src/infrastructure/components/Select/InputSelect.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (Alejoboga20) [reply src/infrastructure/components/Select/InputSelect.tsx]: yes, it's form isAlpha3Code

**Review by Alejoboga20 — APPROVED** [review]

**Pull request merged**